git_clases.rb

```diff
@@ -0,0 +1,107 @@
+require 'net/http'
+require 'json'
+require 'uri'
+# Esta clase es la encargada de crear un Gist, utilizando la API de GitHub.
+# @author Joel Alayon.
+# @since 0.9.24
+class Gist
+    # Esta variable almacena el estado del posteo.
+    # @return [String] estado del posteo.
+    # @example Posteo correcto:
+    #   response_status = "201"
+    # @example Error de conexión:
+    #   response_status = "400"
+    attr_reader :response_status
+    # Esta variable almacena la respuesta del usuario, con respecto a si quiere intentar otra vez.
+    # @note En caso de que haya un error en de conexión, se le preguntará al usuario si quiere intentar otra vez.
+    # @return [String] respuesta del usuario.
+    # @example Si el usuario quiere intentar otra vez:
+    #   try_again = "Si"
+    # @example Si el usuario NO quiere intentar otra vez:
+    #   try_again = "No"
+    attr_reader :try_again
+    # @author Joel Alayon.
+    # Recibe los parametros necesarios de instancia para poder instanciar el objeto.
+    # @note Los parametros del metodo de instancia se los pasa el usuario.
+    
+    def initialize(filename,description,state,content)
```

Comment:
Remember to leave spaces after commas https://github.com/airbnb/ruby#spaces-operators